src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Common.props

```diff
@@ -0,0 +1,15 @@
+﻿<!--
+  WARNING: These targets are intended for building Microsoft's ASP.NET Core repos and are not intended
+  for use outside of Microsoft.
+-->
+<Project>
+  <!-- Set nothing if handled in project file or already done in outer build. -->
+  <PropertyGroup>
+    <EnableApiCheck Condition=" '$(EnableApiCheck)' == '' AND '$(MSBuildRuntimeType)' == 'Core' ">true</EnableApiCheck>
```

Comment:
I disagree. We want developers aware of breaking changes. Many times their unintentionally and we don't catch them until the tool actually runs. We have developers who only build + test in VS.

But I'll let others weigh in to settle this. Should API check cause building in VS to fail? Or should it on fail on build.cmd? cc @Eilon @muratg